### PR TITLE
[frontend] Remove icon action and merge into helper

### PR DESCRIPTION
**Pull request by coolo**
I had hoped you can also remove the load_external_url function, but it's also used for interconnect

### CODE REVIEW

File: src/api/app/helpers/webui/webui_helper.rb [hunk 1]
```diff
@@ -30,11 +30,17 @@ def bugzilla_url(email_list = '', desc = '')
     )
   end
 
-  def user_icon(user, size = 20, css_class = nil, alt = nil)
-    user = User.find_by_login!(user) unless user.is_a? User
-    alt ||= user.realname
+  def user_icon(user, opt = {})
```
Comment (hennevogel) [inline src/api/app/helpers/webui/webui_helper.rb]: This method is used once. How about you drop this too? :-)

Comment (coolo) [reply src/api/app/helpers/webui/webui_helper.rb]: I count 2 usages - once for the large icon on the user's home and once for small icons in a partial

Comment (hennevogel) [reply src/api/app/helpers/webui/webui_helper.rb]: Oh well true, still rather overkill I would say...

**Comment by bgeuken** [comment]
@coolo You could drop the [route](https://github.com/openSUSE/open-build-service/blob/master/src/api/config/routes.rb#L372) as well.

Why did you drop the tests for the webui helper (now user helper)?

**Comment by bgeuken** [comment]
And you can drop the test for the [icon action](https://github.com/opensuse/open-build-service/blob/master/src/api/spec/controllers/webui/user_controller_spec.rb#L342)

**Comment by coolo** [comment]
I just forgot to git add

**Comment by coolo** [comment]
this is work in progress - adding label

**Comment by coolo** [comment]
please review, @eduardoj wants to have a 42.3 package ...

**Pull request merged**